embassy-stm32/src/timer/mod.rs

```diff
@@ -685,6 +680,13 @@ pub trait CaptureCompare16bitInstance:
     + sealed::GeneralPurpose16bitInstance
     + 'static
 {
+    // SimplePwm<'d, T> is implemented for T: CaptureCompare16bitInstance
+    // Advanced timers implement this trait, but the output needs to be
+    // enabled explicitly.
+    // To support basic and advanced timers, this function is added
```

Comment:
It better to say

"To support general-purpose and advanced timer, ..."

since there is already another timer called `BasicTimer`

Reply:
Good point. I changed the comment accordingly